pkg/keyring/keyring.go

```diff
@@ -21,14 +21,17 @@ const (
 // ClearCredentials removes any existing items in the keychain,
 // it will ignore if not found errors
 func ClearCredentials(prefix string) error {
-	for _, k := range []string{username, password, bearer} {
+	for _, k := range []string{username, password, bearer, refreshToken} {
```

Comment:
im not sure we want to remove refreshToken since this make refreshToken obsolete and defeats the purpose